tests/utils/test_time.py

```diff
@@ -0,0 +1,27 @@
+from datetime import datetime, timezone
+
+import pytz
+
+from toolip.utils.time import make_time_aware, now, now_epoch, now_epoch_ms
+
+
+def test_now():
+    assert now().tzinfo == timezone.utc
+
+
+def test_now_epoch():
+    now = datetime.now(timezone.utc).timestamp()
+    assert now_epoch() == int(now)
+    assert type(now_epoch()) == int
```

Comment:
Let's setup mypy instead and use typing on the function.

Reply:
The function being tested already defines the type of returned value `-> int`. I'll just delete the assert.